src/main/scala/io/mesosphere/mesosaurus/Mesosaurus.scala

```diff
@@ -144,6 +145,15 @@ object Mesosaurus extends Logging {
         return if (options.get(name) != null) options.getDouble(name).doubleValue() else defaultValue
     }
 
+    private def parseDouble(s: String, d: Double): Double = {
+        try {
+            s.toDouble
+        }
+        catch {
+            case _ => d
```

Comment:
Bad style -- catch [NonFatal](http://www.scala-lang.org/files/archive/nightly/docs/library/index.html#scala.util.control.NonFatal$) instead.  But `scala.util.Try` will handle that for you as shown above.